src/fsharp/fsc.fs

```diff
@@ -1941,30 +1974,28 @@ let main4 (Args (ctok, tcConfig, errorLogger: ErrorLogger, ilGlobals, ilxMainMod
     DoesNotRequireCompilerThreadTokenAndCouldPossiblyBeMadeConcurrent  ctok
 
     let pdbfile = pdbfile |> Option.map (tcConfig.MakePathAbsolute >> Path.GetFullPath)
-    begin
-        try
-            try 
-                ILBinaryWriter.WriteILBinary 
-                 (outfile, 
-                  { ilg = ilGlobals
-                    pdbfile=pdbfile
-                    emitTailcalls = tcConfig.emitTailcalls
-                    showTimes = tcConfig.showTimes
-                    portablePDB = tcConfig.portablePDB
-                    embeddedPDB = tcConfig.embeddedPDB
-                    embedAllSource = tcConfig.embedAllSource
-                    embedSourceList = tcConfig.embedSourceList
-                    sourceLink = tcConfig.sourceLink
-                    signer = GetStrongNameSigner signingInfo
-                    fixupOverlappingSequencePoints = false
-                    dumpDebugInfo = tcConfig.dumpDebugInfo }, 
-                  ilxMainModule)
-            with Failure msg -> 
-                error(Error(FSComp.SR.fscProblemWritingBinary(outfile, msg), rangeCmdArgs))
-        with e -> 
-            errorRecoveryNoRange e
-            exiter.Exit 1 
-    end
+    try
+        try 
+            ILBinaryWriter.WriteILBinary 
+                (outfile, 
+                { ilg = ilGlobals
+                  pdbfile = pdbfile
+                  emitTailcalls = tcConfig.emitTailcalls
+                  showTimes = tcConfig.showTimes
+                  portablePDB = tcConfig.portablePDB
+                  embeddedPDB = tcConfig.embeddedPDB
+                  embedAllSource = tcConfig.embedAllSource
+                  embedSourceList = tcConfig.embedSourceList
+                  sourceLink = tcConfig.sourceLink
+                  signer = GetStrongNameSigner signingInfo
+                  fixupOverlappingSequencePoints = false
+                  dumpDebugInfo = tcConfig.dumpDebugInfo }, 
+                ilxMainModule)
+        with Failure msg -> 
+            error(Error(FSComp.SR.fscProblemWritingBinary(outfile, msg), rangeCmdArgs))
+    with e -> 
+        errorRecoveryNoRange e
+        exiter.Exit 1 
```

Comment:
He's just changing the indenting in this case  ( @forki - perhaps remove the formatting changes to make the PR minimal? I don't mind either way though)